tests/token_classes.py

```diff
@@ -15,16 +16,16 @@
     scope=','.join(['write_products', 'read_customers']),
     expires_in=86399,
     associated_user_scope=','.join(['read_products']),
-    associated_user={
-        'id': 902541635,
-        'first_name': 'John',
-        'last_name': 'Smith',
-        'email': 'john@example.com',
-        'email_verified': True,
-        'account_owner': True,
-        'locale': 'en',
-        'collaborator': False,
-    },
```

Comment:
There was already an AssociatedUser type defined in the OAuth section, so I reverted the object back to using it instead of a dict